.github/workflows/main-workflow.yml

```diff
@@ -25,5 +25,5 @@ jobs:
     uses: ./.github/workflows/release.yml
     secrets:
       githubToken: ${{ secrets.GITHUB_TOKEN }}
-      npmToken: ${{ secrets.NPM_PUBLISH_TOKEN }}
+      npmToken: ${{ secrets.NPM_CHROMATIC_COM_PUBLISH_TOKEN }}
```

Comment:
Would it make sense to remove the `_COM_` from the secret name since it's a Chromatic publish token?

Reply:
I added that to the name since it is the token for the `@chromatic-com` npm org. Let's keep it for now, and we can always change once we get rid of the other token env vars for the other repos.